packages/tracing/src/browser/metrics.ts

```diff
@@ -187,26 +188,37 @@ export class MetricsInstrumentation {
       }
 
       transaction.setMeasurements(this._measurements);
+      this._tagMetricInfo(transaction);
+    }
+  }
 
-      if (this._lcpEntry) {
-        logger.log('[Measurements] Adding LCP Data');
-        // Capture Properties of the LCP element that contributes to the LCP.
-
-        if (this._lcpEntry.element) {
-          transaction.setTag('lcp.element', htmlTreeAsString(this._lcpEntry.element));
-        }
+  /** Add LCP / CLS data to transaction to allow debugging */
+  private _tagMetricInfo(transaction: Transaction): void {
+    if (this._lcpEntry) {
+      logger.log('[Measurements] Adding LCP Data');
+      // Capture Properties of the LCP element that contributes to the LCP.
 
-        if (this._lcpEntry.id) {
-          transaction.setTag('lcp.id', this._lcpEntry.id);
-        }
+      if (this._lcpEntry.element) {
+        transaction.setTag('lcp.element', htmlTreeAsString(this._lcpEntry.element));
+      }
 
-        if (this._lcpEntry.url) {
-          // Trim URL to the first 200 characters.
-          transaction.setTag('lcp.url', this._lcpEntry.url.trim().slice(0, 200));
-        }
+      if (this._lcpEntry.id) {
+        transaction.setTag('lcp.id', this._lcpEntry.id);
+      }
 
-        transaction.setTag('lcp.size', this._lcpEntry.size);
+      if (this._lcpEntry.url) {
+        // Trim URL to the first 200 characters.
+        transaction.setTag('lcp.url', this._lcpEntry.url.trim().slice(0, 200));
       }
+
+      transaction.setTag('lcp.size', this._lcpEntry.size);
+    }
+
+    if (this._clsEntry) {
+      logger.log('[Measurements] Adding CLS Data');
+      this._clsEntry.sources.map((source, index) =>
```

Comment:
We're seeing some error when running Cypress tests with Chrome in headless mode which I am pretty sure relates to this map. Can `sources` be undefined?

![image](https://user-images.githubusercontent.com/886051/124286441-3f8c7100-db4f-11eb-9923-4710434de51e.png)


![image](https://user-images.githubusercontent.com/886051/124286420-38656300-db4f-11eb-88ab-b1ad6d408373.png)

Reply:
https://github.com/getsentry/sentry-javascript/pull/3775